packages/g/glslang/xmake.lua

```diff
@@ -91,6 +91,9 @@ package("glslang")
             package:add("links", "glslang", "glslang-default-resource-limits")
         end
 
+        os.cp("glslang/MachineIndependent/**.h", path.join(package:installdir("include"), "glslang", "MachineIndependent"))
+        os.cp("glslang/Include/**.h", path.join(package:installdir("include"), "glslang", "Include"))
```

Comment:
```suggestion
        os.mv("glslang/MachineIndependent", path.join(package:installdir("include"), "glslang"))
        os.mv("glslang/Include", path.join(package:installdir("include"), "glslang"))
```
wouldn't moving the folder just work?